.storybook/global.scss

```diff
@@ -1,4 +1,5 @@
+$font-path: '@boston-scientific/anatomy-tokens/lib/fonts';
```

Comment:
should this be defined in storybook files or somewhere in the library, e.g.: `public/styles/abstracts/_vars-type.scss`